cohortextractor/main.py

```diff
@@ -51,28 +88,56 @@ def calculate_measures_results(cohort, input_file):
     yield from measures_manager.calculate_measures()
 
 
-def load_cohort(definition_path):
-    definition_module = load_module(definition_path)
-    imported_classes = [Measure]
-    cohort_classes = [
+def load_cohort_classes(definition_module):
+    return [
         obj
         for name, obj in inspect.getmembers(definition_module)
-        if inspect.isclass(obj) and obj not in imported_classes
+        if inspect.isclass(obj) and obj.__name__ == "Cohort"
     ]
-    assert len(cohort_classes) == 1, "A study definition must contain one class only"
-    return cohort_classes[0]
 
 
-def load_module(definition_path):
-    definition_dir = definition_path.parent
-    module_name = definition_path.stem
+def load_cohort_functions(definition_module):
+    return [
+        obj
+        for name, obj in inspect.getmembers(definition_module)
+        if inspect.isfunction(obj) and obj.__name__ == "cohort"
+    ]
```

Comment:
These now require that you name your class/function `Cohort`/`cohort`; previously when we just had the class, we allowed people to name the class anything, and we just looked for a class in the module.  That did mean that we had to explicitly exclude certain imported classes (e.g. `Measure`) and could mean that a cohort definition breaks if a research imports something unexpected.  I think it's probably acceptable to retrict the class/function name.

Reply:
I think it may even be an improvement to have a fixed name.